WcaOnRails/config/locales/en.yml

```diff
@@ -1168,7 +1168,7 @@ en:
       updated: "Updated registration"
       failed: "Could not update registration"
       deleted: "Successfully deleted your registration for %{comp}"
-      registered: "Registration submitted - make sure to follow payment instructions to avoid disappointment!"
+      registered: "Registration submitted - make check for any payment instructions to avoid disappointment!"
```

Comment:
Typo / grammar mistake

Reply:
I also think it makes sense to explicitly mention Registration Requirements here, because they are such an important and "official" part of how the WCA sanctions competitions

Reply:
I've made a separate in-code suggestion above